client/src/assets/style.css

```
@@ -0,0 +1,3 @@
+.max-width-500{
```

```suggestion
.max-width-500 {
```
put one space before the brackets. 
_Note: one tip to avoid this scenarios is to use a plugin in vscode called prettier then configure it to format on auto save._